bait-files/src/main/java/com/google/sps/servlets/RequestServlet.java

```diff
@@ -63,28 +68,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String notesInput = parameters.get(NOTES)[0];
 
     String countryCode = parameters.get(COUNTRY_CODE)[0];
-    String city = parameters.get("city-input")[0];
-    String addressLine1 = parameters.get("address-line-1-input")[0];
-    String addressLine2 = parameters.get("address-line-2-input")[0];
-
-    String province = "";
-    String postalCode = "";
-    String zipCode = "";
-    String state = "";
+    String city = parameters.get(CITY)[0];
+    String addressLine1 = parameters.get(ADDRESS_1)[0];
+    String addressLine2 = parameters.get(ADDRESS_2)[0];
 
     Address address = new Address();
 
-    if (countryCode == "CA") {
-      postalCode = parameters.get("postal-code-input")[0];
-      province = parameters.get("province-input")[0];
-    } else if (countryCode == "US") {
-      state = parameters.get("state-input")[0];
-      zipCode = parameters.get("zip-code-input")[0];
-    }
-
-    if (countryCode != "IDK") {
-      address = new Address(
-          addressLine1, addressLine2, city, postalCode, zipCode, state, province, countryCode);
+    if (countryCode.equals("CA")) {
+      String postalCode = parameters.get(POSTAL_ZIP)[0];
+      String province = parameters.get(STATE_PROVINCE)[0];
+      address = new Address(addressLine1, addressLine2, city, postalCode, /* zipCode= */ "",
+          /* state= */ "", province, countryCode);
+    } else if (countryCode.equals("US")) {
+      String state = parameters.get(STATE_PROVINCE)[0];
+      String zipCode = parameters.get(POSTAL_ZIP)[0];
+      address = new Address(addressLine1, addressLine2, city, /* postalCode= */ "", zipCode, state,
```

Comment:
This might need a formatter (or if it formats it really strangely bc of the comments, just take them out)